inception-imls-external/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/ExternalRecommender.java

```diff
@@ -190,9 +191,11 @@ private String serializeTypeSystem(CAS aCas) throws RecommendationException
 
     private String serializeCas(CAS aCas) throws RecommendationException
     {
-        try (ByteArrayOutputStream out = new ByteArrayOutputStream()) {
-            XmiCasSerializer.serialize(aCas, null, out, true, null);
-            return new String(out.toByteArray(), UTF_8);
+        try (StringWriter out = new StringWriter()) {
+            XmiCasSerializer xmiCasSerializer = new XmiCasSerializer(null);
```

Comment:
I would add a comment here that we do not limit the serialization by the type system, the `null` else requires to read the UIMA code.